contracts/0.0.4/chatroom-slim.js

```diff
@@ -420,12 +420,16 @@ ${this.getErrorInfo()}`;
     const rootGetters = (0, import_sbp3.default)("state/vuex/getters");
     const groupID = rootGetters.groupIdFromChatRoomId(contractID);
     const path = `/group-chat/${contractID}`;
-    makeNotification({
-      title: `# ${chatRoomName}`,
-      body: text,
-      icon: rootGetters.globalProfile2(groupID, username).picture,
-      path
-    });
+    const identityState = rootGetters.globalProfile2(groupID, username);
+    if (identityState) {
+      makeNotification({
+        title: `# ${chatRoomName}`,
+        body: text,
+        icon: identityState.picture,
+        path
+      });
+    } else {
+    }
```

Comment:
PRs must never modify existing pinned contracts. They can only ever pin new ones.

None of the files under `0.0.4` can change, so please remove all of these changes.